src/components/LineChart/LineChart.jsx

```diff
@@ -371,12 +380,14 @@ const LineChart = createClass({
 			yAxisTitleColor,
 			yAxisMin,
 			yAxisTooltipFormatter,
+			yAxisTooltipDataFormatter,
```

Comment:
Gotcha. Adding a default value of `yScale.tickFormat()` means we have to move the variable definition (for `yScale`) above the props. Cool with that?